passive/find_reflected_params.py

```diff
@@ -0,0 +1,46 @@
+"""
+
+looks for parameter values that are reflected in the response.
+Author: maradrianbelen.com
+
+The scan function will be called for request/response made via ZAP, excluding some of the automated tools
+Passive scan rules should not make any requests 
+
+Note that new passive scripts will initially be disabled
+Right click the script in the Scripts tree and select "enable"
+
+"""  
+def compare(paramvalue_pair,msg): 
+	value=paramvalue_pair.split('=')
+	body=msg.getResponseBody().toString()
+	header=msg.getResponseHeader().toString();
+	reflected=''
+	if (value[1] in body or value[1] in header) and value[1] != '': 
```

Comment:
This will still lead to errors:
```
Traceback (most recent call last):
  File "<script>", line 34, in scan
  File "<script>", line 18, in compare
IndexError: index out of range: 1
```
this can be reproduced with a query parameter without a value, e.g.: `x=y&z`.